src/odemis/gui/cont/tabs.py

```diff
@@ -2825,9 +2829,28 @@ def _onSEMpxs(self, pixel_size):
         best_mpp = self._sem_view.mpp.clip(best_mpp)
         self._sem_view.mpp.value = best_mpp
 
+    def _onAlignPos(self, pos):
+        """
+        Called when the aligner is moved (and the tab is shown)
+        :param pos: (dict str->float) updated position of the aligner
+        """
+        # Return if aligner doesn't have MD_FAV_POS_ACTIVE and MD_FAV_POS_DEACTIVE metadata
+        md = self._aligner_xy.getMetadata()
+        if model.MD_FAV_POS_ACTIVE not in md and model.MD_FAV_POS_DEACTIVE in md:
+            return
+        # Check if updated position is close to FAV_POS_DEACTIVE
+        for axis in self._aligner_xy.axes:
+            if not _isNearDeactive(pos, md[model.MD_FAV_POS_DEACTIVE], axis):
```

Comment:
Please don't use private methods.
I'd actually not recommend using this method anyway.
You could just write a check like this:
``` python
dist_deactive = math.hypot(pos["x"] - MD_FAV_POS_DEACTIVE["x"],
               pos["y"] - MD_FAV_POS_DEACTIVE["y"])
if dist_deactive <= 0.1e-3: # within 0.1mm of deactive is considered deactivated.
       ...warning...
      return
```